src/program.js

```diff
@@ -436,8 +436,8 @@ Example: $0 --help run.
     .command('run', 'Run the extension', commands.run, {
       'target': {
         alias: 't',
-        describe: 'The extensions runners to enable (e.g. firefox-desktop, ' +
-                  'firefox-android). Specify this option multiple times to ' +
+        describe: 'The extensions runners to enable. ' +
```

Comment:
Could you combine this line with the next two lines so that the string is only cut when the 80-character line limit has been reached?

Reply:
OK